api/admin/routes.py

```diff
@@ -46,9 +46,10 @@ def decorated(*args, **kwargs):
 def requires_csrf_token(f):
     @wraps(f)
     def decorated(*args, **kwargs):
-        token = app.manager.admin_sign_in_controller.check_csrf_token()
-        if isinstance(token, ProblemDetail):
-            return token
+        if flask.request.method in ["POST", "PUT", "DELETE"]:
```

Comment:
I don't understand much about CRSF so it's not clear to me why GET requests don't need a token.